src/Config.php

```diff
@@ -1402,6 +1402,12 @@ function formatGlobalLayout(layout) {
             echo "</tr></table>";
 
             echo "</td></tr>";
+
+            echo "<tr class='tab_bg_2'>";
+            echo "<td><label for='dropdown_use_flat_dropdowntree_on_display$rand'>" . __('Display the tree dropdown complete name on display') . "</label></td><td>";
```

Comment:
The option label should be more precise and clearly explain that it targets search results (= list display) only.
Also avoid "Display... on display" wording.

Reply:
renamed to ```Display the tree dropdown complete name on search result```

Reply:
rename to ```Display the tree dropdown short name on search result```